### ISSUE TITLE
Rpm package extracted to file with backslash in it's name

### ISSUE THREAD

**Issue by decmaxn** [git-town/git-town#1629]
Hello,  not sure why it is packaged this way, is it better to be forward slashes?
```
$ sudo rpm -qlp git-town_7.4.0_linux_intel_64.rpm
\usr\local\bin\git-town
$ ls *git*
git-town_7.4.0_linux_intel_64.rpm    \usr\local\bin\git-town
```
Best Regards
Victor Ma

**Comment by kevgo** [comment]
Agree this is a bug, probably because I run the [release script](https://github.com/git-town/git-town/blob/master/Makefile#L52-L70) on Windows in order to build the Windows installer, and goreleaser doesn't handle that properly when building the RPM package.

**Comment by kevgo** [comment]
Same issue with the `.rpm` package for ARM.

**Comment by kevgo** [comment]
The issue doesn't happen when running goreleaser on Linux.

**Comment by kevgo** [comment]
@decmaxn I have uploaded fresh RPM packages that fix the broken path to release 7.4.0. Can you please verify that they install correctly?

**Comment by decmaxn** [comment]
@kevgo I just tested the new RPM package and it works now, thank you very much.
